settings.ini

```diff
@@ -14,6 +14,10 @@ MODEM_SERIAL3=/dev/ttyUSB3
 [device_rpi]
 ARCH=arm64
 I2C_DEVICE=/dev/i2c-1
+MODEM_SERIAL0=/dev/ttyUSB0
```

Comment:
having them is safe at this moment. Whoever removed them didn't remove from docker-compose.template. Keeping for the time being.